allure-report-face/src/components/pane-set/styles.css

```diff
@@ -64,3 +63,7 @@
     }
   }
 }
+
+.pane + .pane {
+  box-shadow: 0 0 3px #eee;
+}
```

Comment:
I don't see the border:

![image](https://cloud.githubusercontent.com/assets/812240/15475728/c3399bf8-210b-11e6-898e-1622f08fe58d.png)